zaza/openstack/charm_tests/ceph/tests.py

```diff
@@ -23,6 +23,8 @@
 )
 import requests
 import tempfile
+import boto3
```

Comment:
I went ahead with boto3 because I think it is already being used. The [requirements.txt](https://github.com/openstack-charmers/zaza-openstack-tests/blob/66d0c9237c792d286e713921a8d3ec298b15c53b/requirements.txt#L8) has it and it is used in Glance, Gnocchi, Swift and Trillio scripts (setup.py or tests.py). Can you please confirm once whether using boto3 would require dependency change as well ? Otherwise I will switch to using requests and crafting rest api calls.

Reply:
Oh, I wonder how that snuck in!  Please could you do me a favor and also pin it as part of this PR?